doc/classes/LineEdit.xml

```diff
@@ -180,6 +194,13 @@
 				Selects the whole [String].
 			</description>
 		</method>
+		<method name="set_focus_edit">
+			<return type="void" />
+			<param index="0" name="focus_edit" type="bool" />
+			<description>
+				Toggles focus edit mode, if [member editable] is [code]false[/code] it will just focus the [LineEdit].
```

Comment:
This description sounds misleading. It should more clearly say that it affects behavior of `grab_focus()`.

Reply:
`grab_focus_edit()` is an alias for 
```cpp
LineEdit::grab_focus();
LineEdit::set_focus_edit(true);
LineEdit::select_all(); // optionally.
```
This is similar to the old behavior of `grab_focus()`, and yes `set_focus_edit(true)` is the same as `grab_focus_edit(false)` so maybe we can only expose grab_focus_edit() because it's more easier to notice when we type `grab_focus`. and maybe we can add release_focus_edit() to set `focus_edit` to false.

Reply:
Yeah that sounds like what I suggested below, but with different names.